practice-project-api/routes/calculateRoutes.ts

```diff
@@ -0,0 +1,34 @@
+import { Router, Request, Response } from "express";
+
+export const calculateRoute = Router();
+
+calculateRoute.post("/calculate", (req: Request, res: Response, next) => {
+
+  type calculatorOperation = '*' | '/' | '+' | '-';
+
+  interface Calculate {
+    firstNumber: number,
+    secondNumber: number,
+    operation: calculatorOperation,
+  };
+
+  const input: Calculate = {
+    firstNumber: req.body.firstNumber,
```

Comment:
It's redundant to add the type into the name of a variable, usually. `firstNumber` is a `number` type so it's kinda pointless to have `Number` in the name.